src/components/record/Controls.tsx

```diff
@@ -0,0 +1,89 @@
+import StopIcon from "../../icons/stop";
+import CameraOnIcon from "../../icons/camera-on";
+import CameraOffIcon from "../../icons/camera-off";
+import RecordIcon from "../../icons/record";
+import MicIcon from "../../icons/mic";
+import ShareIcon from "../../icons/share";
+import CancelIcon from "../../icons/cancel";
+import DownloadIcon from "../../icons/download";
+import FeedbackIcon from "../../icons/feedback"; // AI Feedback 아이콘 추가
```

Comment:
```suggestion
import { StopIcon, CameraOnIcon, CameraOffIcon, RecordIcon, MicIcon, ShareIcon, CancelIcon, DownloadIcon, FeedbackIcon } from "../../icons";
```

위처럼 바꾸는건 어떨까요..?! "../../icons"가 모두 통일되는 것 같은데, import를 줄마다 적는 것보다는 가독성이 좋아질 것 같아서요!

Reply:
이렇게 축약하는거 불가능하다네요..ㅠ